oci/provider.go

```diff
@@ -500,7 +501,7 @@ func buildConfigureClientFn(configProvider oci_common.ConfigurationProvider, htt
 		terraformCLIVersion = ociProvider.TerraformVersion
 	}
 	userAgentProviderName := getEnvSettingWithDefault(userAgentProviderNameEnv, defaultUserAgentProviderName)
-	userAgent := fmt.Sprintf(userAgentFormatter, oci_common.Version(), runtime.Version(), runtime.GOOS, runtime.GOARCH, terraform.VersionString(), terraformCLIVersion, userAgentProviderName, Version)
+	userAgent := fmt.Sprintf(userAgentFormatter, oci_common.Version(), runtime.Version(), runtime.GOOS, runtime.GOARCH, sdkMeta.SDKVersionString(), terraformCLIVersion, userAgentProviderName, Version)
```

Comment:
It may be easier to just drop the SDK version since its implied by the provider version.